app/src/routes/rootRoute.jsx

```diff
@@ -17,13 +17,17 @@ import SettingsPage from 'pages/inside/settingsPage/settingsPage';
 
 import { LoginPage } from 'pages/outside/loginPage/loginPage';
 
+import { LocalizationSwitcher } from 'components/main/localizationSwitcher';
+
 const LoginRoute = () => (
   <EmptyLayout>
+    <LocalizationSwitcher />
     <LoginPage />
   </EmptyLayout>
 );
 const AppRoute = () => (
   <AppLayout>
+    <LocalizationSwitcher />
```

Comment:
What about admin route?

Reply:
this is only for the test, in the future this component will be deleted. But in the admin area, too, I'll add